integration-tests/wallet-manager-scenario.spec.ts

```diff
@@ -5,12 +5,12 @@ import { MANAGER_LAMBDA, Protocols } from "@taquito/taquito";
 CONFIGS().forEach(({ lib, rpc, setup, knownBaker, knownContract, protocol }) => {
   const Tezos = lib;
 
-  describe(`Manager TZ: ${rpc}`, () => {
+  describe(`Test TZ Manager through wallet api: ${rpc}`, () => {
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    test('tests manager transfer scenarios with wallet APi contract', async (done) => {
+    test('Verify wallet.transfer scenarios: implicit transfer to and from contracts, set and remove delegate, and transfer from a contract to a contract.', async (done: () => void) => {
```

Comment:
`: () => void` is not needed without jest retry

Reply:
removed